features/storeHealthCertificate.feature

```diff
@@ -8,9 +8,9 @@ Feature: Store Health Certificate
       And I set json body to the file at ./features/fixtures/storeHealthCertificate/Bundle-DDCCDocument.json
     When I receive a response
     Then I expect response should have a status 201
-      And I expect response header location should be /DDCCDocument/TESTID
-      And I store response at /DDCCDocument/TESTID as Bundle-DDCCDocument
-
+      And I expect response header location should be /Bundle/TESTID
```

Comment:
I removed the step "I expect response header location should be..." since it hardcodes the path. Do we need to include a step that checks the format of the response header location (that it includes the type/id) by instead pulling it from the response location and then checking its format?

Reply:
That should be fine.